src/reference/interactive/index.pug

```diff
@@ -17,47 +17,49 @@ block reference
+    p A Game Client is software code which processes interactive events. It is written by developers who want to create an interactive experience. Game Clients connect to the #[a(href='#the-interactive-service') Interactive Service], listens to events and updates sent from the service to take action within their environment, thus affecting the broadcast.
 
-    p Game Clients can run in any environment, including:
+    p You can add Game Client code as part of various environments, including:
     ul
-        li In a game's existing code
-        li In a 3rd party mod for a game
-        li In a standalone application
-        li In a game's server-side service
+        li Game's current codebase
+        li Third party mod of a game
+        li Standalone application
+        li Game's server-side service
 
     h3#the-interactive-service The Interactive Service
 
-    p The Interactive Service is a service operated by Beam. A Game Client connects to it to create an interactive session. Once a session is established, the service acts as a mediator for the Interactive session. The service manages data flow and state within the session by processing and distributing data emitted from both the #[a(href='#the-gameclient') Game Client] and #[a(href='#participants') Participants]. To interact with the session or update any state in the session a Game Client has to send messages to the service.
+    p The Interactive Service is a service operated by Beam. A Game Client connects to it to create an interactive session. Once a session is established, the service acts as a mediator for the Interactive session. The service manages data flow and state within the session by processing and distributing data sent to and from both the #[a(href='#the-gameclient') Game Client] and #[a(href='#participants') Participants]. The Game Client sends messages to the service when it needs to update the session state or to interact with the session. 
 
     h3#an-interactive-project An Interactive Project
 
-    p An Interactive Project is created by a developer or team on Beam within our #[a(href='https://beam.pro/i/studio' target='_blank') Interactive Studio]. It stores settings and metadata about your interactive experience. Projects are owned by a single Beam user but can be #[a(href='#sharing-your-project') shared] with other users.
+    p The Interactive Project stores settings and metadata about your interactive experience. It is created by developers through the #[a(href='https://beam.pro/i/studio' target='_blank') Beam Interactive Studio]. Projects are owned by a single Beam user but can be #[a(href='#sharing-your-project') shared] with other users.
```

Comment:
Definite article sounds incorrect here. (the)

We're being general here and talking about an indefinite concept.